src/webextension/list/components/item-summary.css

```diff
@@ -3,23 +3,19 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 .item-summary {
-  padding: 0.5em;
+  padding: 1em 1.3em;
```

Comment:
While we're changing this, perhaps it should use `px`? Maybe @changecourse has an opinion...